grails-app/controllers/org/broadinstitute/orsp/UrlMappings.groovy

```diff
@@ -50,6 +50,9 @@ class UrlMappings {
         '/api/consent-groups'(controller: 'consentGroup', action: 'projectConsentGroups', method: 'GET')
         '/api/consent-group/upload-modal'(controller: 'consentGroup', action: 'loadModalWindow', method: 'GET')
 
+        '/api/break-link'(controller: 'consentGroup', action: 'breakLink', method: 'DELETE')
```

Comment:
Is the 'DELETE' accurate here? You're reading params which means there is a post-body content. This seems more like a POST than a DELETE.